AutocompleteClient/FW/API/Parser/Quizzes/AbstractQuizzesResponseParser.swift

```diff
@@ -0,0 +1,14 @@
+//
+//  AbstractQuizzesResponseParser.swift
+//  AutocompleteClient
+//
+//  Copyright © Constructor.io. All rights reserved.
+//  http://constructor.io/
+//
+
+import Foundation
+
+protocol AbstractQuizzesResponseParser {
```

Comment:
=> `AbstractQuizResponseParser`

We might need two different parsers for each endpoint now 🤔